content/en/docs/setup/learning-environment/_index.md

```diff
@@ -31,5 +31,20 @@ Kubernetes, or for daily development work.
 You can follow the official
 [Get Started!](https://minikube.sigs.k8s.io/docs/start/) guide if your focus is
 on getting the tool installed.
--->
+
+<a class="btn btn-primary" href="https://minikube.sigs.k8s.io/docs/start/" role="button" aria-label="View minikube Get Started! Guide">View minikube Get Started! Guide</a>
+
+Once you have `minikube` working, you can use it to
+[run a sample application](/docs/tutorials/hello-minikube/).
+
+
+## {{% heading "whatsnext" %}}
+
+- Make sure to [install `kubectl`](/docs/tasks/tools/#kubectl) as well.
```

Comment:
We should make setting up `kubectl` be a key step; it's more than a “what's next” item. IMO.

Reply:
People always had issues with that step, which is why `minikube kubectl` was added for them.